lms/djangoapps/commerce/admin.py

```diff
@@ -0,0 +1,8 @@
+""" Admin site bindings for commerce app. """
+
+from django.contrib import admin
+
+from commerce.models import CommerceConfiguration
+from config_models.admin import ConfigurationModelAdmin
+
+admin.site.register(CommerceConfiguration, ConfigurationModelAdmin)
```

Comment:
Extend this and update the list view to display the current values. Our current list views are not useful. Bonus points if you can update `ConfigurationModelAdmin` to make this happen for all configurations.